Commands/guessgame.py

```
@@ -0,0 +1,190 @@
+            if self.hintTimer:
+                self.hintTimer.cancel()
+
+            # user guessed emote right, move to next round
+            self.send_privmsg(
+                message['command']['channel'], f"{message['tags']['display-name']} guessed it right! It's {currentEmote}")
+
+            time.sleep(1.1)
+            if self.currentRound + 1 == self.numRounds:
+                # end the game
+                self.send_privmsg(
+                    message['command']['channel'], f"Game has ended.")
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)